tests/default/indices/validate_query.yaml

```diff
@@ -0,0 +1,49 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
```

Comment:
This file should be `validate/query.yaml`.

Reply:
no worries! i'm really interested in working with OpenSearch in the form of tests or writing specs to understand it more deeply.